apps/test/netsim/NetSimVizNode.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+/* global describe */
+/* global beforeEach */
+/* global it */
+/* global $ */
+
+var testUtils = require('../util/testUtils');
+testUtils.setupLocale('netsim');
+var assert = testUtils.assert;
+var assertEqual = testUtils.assertEqual;
+var netsimTestUtils = require('../util/netsimTestUtils');
+var fakeShard = netsimTestUtils.fakeShard;
+var assertTableSize = netsimTestUtils.assertTableSize;
+
+var NetSimVizNode = require('@cdo/apps/netsim/NetSimVizNode');
+
+describe("NetSimVizNode", function () {
```

Comment:
Experimentally, I added this test thinking it's probably a whole lot better to just have a test that loads the file and runs the default constructor than no test at all.  Agree, disagree?  Should I go back and do this for all of my visualization elements, which are untested right now?